### ISSUE TITLE
Clarification on: "Our network performed no better than random"

### ISSUE THREAD

**Issue by SohamTamba** [untitled-ai/self_supervised#5]
In your blog, you said "We originally implemented BYOL in PyTorch using code we had written for MoCo. When we began training our network, we found that our network performed no better than random". 
Just to clarify, do you mean to say that a randomly intialised network without any training performs the same as BYOL without Batchnorm? 
If yes, isn't that strange? I would expect a random network to perform better than a mode collapsed network.

Also, thank you for the blog. I enjoyed reading it :smiley:.

**Comment by abefetterman** [comment]
Thanks @SohamTamba , this is a good question!

In the early stages of representation collapse (in our experiments, until cosine similarity is around 0.99999), the linear network used for accuracy evaluation is capable of using the small variation between embeddings just as well as the original random embeddings. This results in evaluation performance that is flat for a while. As collapse continues past that, the variation in encodings is no longer high enough for the linear fit to separate instances from each other, and the performance drops down to random chance. You can see this in [Appendix C](https://untitled-ai.github.io/appendix-for-understanding-self-supervised-contrastive-learning.html#appendix-c) under "Experiments without batch normalization in the encoder."

Let us know if you have any more questions!
